Week_4/Readme.md

```diff
@@ -0,0 +1,128 @@
+## Day 1
+Date: 1/7/2020 (Tuesday)
+
+## Lesson: Conditional Execution
+
+#### Goals:
+* To understand what an algorithm is
+* To understand the value of conditional execution
+* To understand how to test for conditions
+* To understand if and if-else statements
+* To understand Python's if-else syntax and indentation requirements
+
+#### Algorithms
+* How do you define a process or solution to a problem?
+* How do you make a peanut butter and jelly sandwich?
+* An algorithm is a set of instructions designed to achieve a certain goal or solve a certain problem
+* We all use algorithms every day of our lives, we just may not call them that
+  1. How do you get ready for school in the morning?
+  2. How do you make your bed?
+  3. How do you make a Peanut Butter Sandwich?
+
+#### Limitations of Linear Program Flow
+* So far the instructions you have given the computer have been linear sequences of simple instructions
+* All instructions executed once and once only
+* No chance to deal with problems that arise
+* Using PB&J algorithm from above, discuss the drawbacks of having one fixed set of instrctions that happen once each
+  * What if the bag of bread is empty?
+  * How do you know when to stop putting PB on the bread?
+  * What if you stop after one scoop of jelly? Is it enough?
+  * What if you run out of Peanut Butter?
+  * What if you have more than one kind of jelly?
+
+#### Logical Operators
+* Have already covered Boolean type (TRUE/FALSE)
+* `<, >, ==, !, etc`
+* `||, &&, ()`
```

Comment:
Python uses `or` and `and` for logical operators.

Reply:
I'll be covering some of this with expressions in week 2.